File: ABC/ABC267/A.cpp
#include<iostream>
#include<string>
using namespace std;

int main(void) {
  string s; cin >> s;
  int remain;
  if(s == "Monday") remain=5;
  else if(s == "Tuesday") remain=4;
  else if(s == "Wednesday") remain=3;
  else if(s == "Thursday") remain=2;
  else if(s == "Friday") remain=1;
  cout << remain << endl;
  return 0;
}

File: ABC/ABC267/A.java
import java.util.*;

public class Main {
   public static void main(String args[]) {
     Scanner scanner = new Scanner(System.in);
     String s = scanner.nextLine();
     List list = new ArrayList<>(Arrays.asList("Friday","Thursday","Wednesday","Tuesday","Monday"));
     int ans;
     ans = list.indexOf(s) + 1;
     System.out.println(ans);
   }
}